definitions/pipelines/immuno.cwl

```diff
@@ -0,0 +1,777 @@
+    phase_vcf:
+        run: ../subworkflows/phase_vcf.cwl
+        in:
+            somatic_vcf: somatic/final_vcf
+            germline_vcf: germline/final_vcf
+            reference: reference
+            reference_dict: reference_dict
+            bam: somatic/tumor_cram
+        out:
+            [phased_vcf]
+    extract_alleles:
+        run: ../tools/extract_hla_alleles.cwl
+        in:
+            allele_file: germline/optitype_tsv
+        out:
+            [allele_string]
+    pvacseq:
+        run: ../subworkflows/pvacseq.cwl
+        in:
+            detect_variants_vcf: somatic/final_vcf
+            sample_name: 
+                default: 'TUMOR'
+            normal_sample_name: 
+                default: 'NORMAL'
+            rnaseq_bam: rnaseq/final_bam
+            reference_fasta: reference
+            readcount_minimum_base_quality: readcount_minimum_base_quality #should this be different from qc_minimum_base_quality?
+            readcount_minimum_mapping_quality: readcount_minimum_mapping_quality #should this be different from qc_minimum_mapping_quality
+            gene_expression_file: rnaseq/stringtie_gene_expression_tsv #not sure if this connection is correct
+            transcript_expression_file: rnaseq/transcript_abundance_tsv #not sure if this connection is correct
+            alleles: extract_alleles/allele_string
+            prediction_algorithms: prediction_algorithms
+            epitope_lengths: epitope_lengths
+            binding_threshold: binding_threshold
+            allele_specific_binding_thresholds: allele_specific_binding_thresholds
+            minimum_fold_change: minimum_fold_change
+            peptide_sequence_length: peptide_sequence_length
+            top_score_metric: top_score_metric
+            additional_report_columns: additional_report_columns
+            fasta_size: fasta_size
+            downstream_sequence_length: downstream_sequence_length
+            exclude_nas: exclude_nas
+            phased_proximal_variants_vcf: phase_vcf/phased_vcf
+            maximum_transcript_support_level: maximum_transcript_support_level
+            normal_cov: normal_cov
+            tdna_cov: tdna_cov
+            trna_cov: trna_cov
+            normal_vaf: normal_vaf
+            tdna_vaf: tdna_vaf
+            trna_vaf: trna_vaf
+            expn_val: expn_val
+            net_chop_method: net_chop_method
+            net_chop_threshold: net_chop_threshold
+            netmhc_stab: netmhc_stab
+            n_threads: pvacseq_threads
+            variants_to_table_fields: variants_to_table_fields #the defaults set in this file will override the defaults for this parameter set in the pvacseq subworkflow; is this ok?
```

Comment:
I think we'd want to use consistent output fields between the different variants to table steps so overwriting the defaults is ok.